packages/analytics-js-integrations/src/integrations/TiktokAds/browser.js

```diff
@@ -90,14 +95,14 @@ class TiktokAds {
           });
         }
       });
-    } else {
-      event = eventNameMapping[event];
-      const updatedProperties = getTrackResponse(message);
-      window.ttq.track(event, updatedProperties);
+      return;
     }
+    event = eventNameMapping[event] || event;
+    const updatedProperties = getTrackResponse(message);
+    window.ttq.track(event, updatedProperties);
   }
 
-  page(rudderElement) {
+  page() {
```

Comment:
can you explain this, I know we are not using this param but, is there any common function signature for page, track etc to take rudderElement?

Reply:
Yeah there is let me revert this change

Reply:
reverted